install/victoriametrics-install.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing VictoriaMetrics"
+RELEASE=$(curl -s https://api.github.com/repos/VictoriaMetrics/VictoriaMetrics/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+mkdir -p /etc/victoriametrics
+mkdir -p /var/lib/victoriametrics
+groupadd -r victoriametrics
+useradd -g victoriametrics -d /var/lib/victoriametrics -s /sbin/nologin --system victoriametrics
+chown -R victoriametrics:victoriametrics /var/lib/victoria-metrics
```

Comment:
Is the user needed or can it be run with root?

Reply:
It can be run as root but I think that is better to run it by an own user.

Reply:
You run it at root anyway, the user it not used. You can remove it then.